integration/src/main/java/scripts/testscripts/WorkspaceLifecycle.java

```diff
@@ -21,7 +21,7 @@
 public class WorkspaceLifecycle extends WorkspaceApiTestScriptBase {
   private static final Logger logger = LoggerFactory.getLogger(WorkspaceLifecycle.class);
   // Perf tests run same test in different threads. Use uuid to make userFacingIds unique.
-  String uuid = UUID.randomUUID().toString();
+  private static final String uuid = UUID.randomUUID().toString();
```

Comment:
nit: these should all be UPPER_SNAKE_CASE.

Reply:
Done. Left `uuid` since `UUID` is taken.